docs/README.md

```diff
@@ -0,0 +1,38 @@
+# Kurtosis CDK documentation
+
+Welcome to the Kurtosis docs.
+
+## Running locally
+
+### Prerequisites
+
+1. [Python 3.12](https://www.python.org/downloads/).
+2. [`virtualenv`](https://pypi.org/project/virtualenv/): Install using `pip3 install virtualenv`.
+
+### Setup
+
+1. Clone the repository.
+2. `cd` to the root.
+3. Run the `run.sh` script. You may need to make the script executable: `chmod +x run.sh`
+
+```sh
+./run.sh
+```
+
+The site comes up at http://127.0.0.1:8000/ 
+
+## Style guide
+
+We are using the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/).
+
+## Contributing
+
+1. Add your draft documentation to `doc-drafts` and test it there.
+2. When you're happy with it, raise a ticket on our [slack channel](https://slack.com/shortcuts/Ft0613JADARL/ea28ee64d7fcc12fd40b0eb310b583d3), or [JIRA board](https://polygon.atlassian.net/jira/software/projects/DOCS/boards/79) to let us know, and we'll incorporate it into the docs.
```

Comment:
I would put this under a `Contributing` section and rather give a quick explanation of what is the kurtosis-cdk package, how you can quickly get started (e.g. give a few commands) and give a quick overview of the architecture. What do you think? :)

Reply:
Hi @leovct actually, this page doesn't get published when we import it so it's really for local contributors to the docs on your repo.

These docs will sit directly in the CDK get started section when I import it later on..

![kurtosis-import](https://github.com/0xPolygon/kurtosis-cdk/assets/149679879/5bc98a0c-bb7b-4320-bab2-096422332dcf)

So, a quick explanation should go in the overview page instead and yes we can do that :)

Reply:
Oh ok makes sense! :)